### ISSUE TITLE
doesn't work properly on deviantart

### ISSUE THREAD

**Issue by MozarteanChaos** [shinigami-eyes/shinigami-eyes#57]
something about the way deviantart does page links makes it extremely difficult (if not outright impossible) to mark a user as anti-trans - when right-clicking the link, the option simply doesn't appear, and the same goes for trans-friendly. no idea if this can be fixed on the extension's end, but if it could then that'd be cool

**Comment by ShieldOfWesnoth** [comment]
Agreed!  I encountered this issue a couple years back, myself, as well.  DeviantART may not be a news site, but it's still got plenty of social elements, and it'd be nice to be able to avoid supporting transphobes on art-sharing platforms; especially considering the large amounts of anti-trans artworks created by transphobes even on websites not purely intended for art publication.
